models/load-test.go

```diff
@@ -125,8 +125,6 @@ func (m *MesheryResult) ConvertToSpec() (*BenchmarkSpec, error) {
 		results = httpResults
 		logrus.Debugf("httpresults: %+v", httpResults)
 		b.EndpointURL = httpResults.URL
-	} else {
```

Comment:
We do want to add gRPC support. If this else statement is harmful in the meantime, we can remove it.

Reply:
We can add it later onwards. No sense of having a blank else block.